src/main/java/com/powsybl/network/conversion/server/NetworkConversionController.java

```diff
@@ -85,14 +84,40 @@ public ResponseEntity<byte[]> exportNetwork(@Parameter(description = "Network UU
                                                 @Parameter(description = "Variant Id") @RequestParam(name = "variantId", required = false) String variantId,
                                                 @Parameter(description = "File name") @RequestParam(name = "fileName", required = false) String fileName,
                                                 @org.springframework.web.bind.annotation.RequestBody(required = false) Map<String, Object> formatParameters
-                                                ) throws IOException {
+                                                ) {
         LOGGER.debug("Exporting network {}...", networkUuid);
         ExportNetworkInfos exportNetworkInfos = networkConversionObserver.observeExport(format, () -> networkConversionService.exportNetwork(networkUuid, variantId, fileName, format, formatParameters));
         HttpHeaders header = new HttpHeaders();
         header.setContentDisposition(ContentDisposition.builder("attachment").filename(exportNetworkInfos.getNetworkName(), StandardCharsets.UTF_8).build());
         return ResponseEntity.ok().headers(header).contentType(MediaType.APPLICATION_OCTET_STREAM).body(exportNetworkInfos.getNetworkData());
     }
 
+    @PostMapping(value = "/cases/{caseUuid}/convert/{format}")
```

Comment:
```suggestion
    @PostMapping(value = "/cases/{caseUuid}/export/{format}")
```
?
for consistency with other endpoint?

Reply:
I think convert is more explicite